### PR TITLE
Add trail's project files

### PR DESCRIPTION
**Pull request by pratap-panabaka**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <a href="#welcome-section">Welcome Section</a>
+    </nav>
+    <style type="text/css">
+      @media only screen and (max-width: 600px) {
+        #welcome-section {
+          background-color: #000000;
+          color: #ffffff;
+        }
+      }
+    </style>
```
Comment (pratap-panabaka) [inline index.html:29]: It would be better if you keep your style definition in a separate CSS file instead of inline styles.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+    <section id="welcome-section">
+      <h1>Hi! I am a Software Developer</h1>
+    </section>
+    <section id="projects">
+      <a id="profile-link" class="project-tile" href="" target="_blank"><img class="project-image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" "title=" project"></img>
```
Comment (pratap-panabaka) [inline index.html:37]: could you please add an "alt" attribute on the line number 36 for img tag? The alt attribute provides alternative information for an image if a user for some reason cannot view it.

**Review by pratap-panabaka — COMMENTED** [review]
# Changes Required :recycle: 

Please check the comments below and make the necessary changes and submit for re-review.

- could you please configure linters, rite now there are no linters configured and it is not possible to check for [linters errors](https://github.com/microverseinc/linters-config)
- It would be better if you add more details in the README.md file according to the guidelines given here [README.md](https://github.com/microverseinc/readme-template)

Thanks :smile:

**Review by KayLemba — APPROVED** [review]
Hi @PRATAP-KUMAR  😃, 

You did a great job delivering such an amazing review 💯 Great summary 👍 Well done using the GitHub inline comments feature 👍 Polite and well-explained comments 👍

Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**